asset_classes/player.py

```diff
@@ -0,0 +1,10 @@
+import pygame
+
+from asset_classes.entity import Entity
+
+
+class Player(Entity):
+    def __init__(self, name, img, width, height, angle, hp):
+        super().__init__(name, img, width, height, angle)
+
+        self.hp = hp
```

Comment:
All the work in Entity makes creating players super simple. The same for enemies, levels, items... 

So a Player is an Entity which is a Sprite, but with an additional HP property.